plugins/pulp_python/plugins/importers/sync.py

```diff
@@ -0,0 +1,267 @@
+"""
+This module contains the necessary means for a necessary means for syncing packages from PyPI.
```

Comment:
On 02/20/2015 06:08 PM, Austin wrote:
>> +This module contains the necessary means for a necessary means for syncing packages from PyPI.
> 
> s/a necessary means for//

Dude, haven't you seen The Big Lebowski?